src/config.zig

```diff
@@ -381,6 +381,9 @@ comptime {
     assert(lsm_batch_multiple % 2 == 0);
 
     // SetAssociativeCache requires a power-of-two cardinality.
+
+    // TODO These are now overridable from CLI, so we need to check the
+    // invariant at compile time somewhere.
```

Comment:
These cache counts end up being checked by the `SetAssociativeCache`: https://github.com/tigerbeetledb/tigerbeetle/blob/d15acc663f8882cb02413129e8351bf3238335e6/src/lsm/set_associative_cache.zig#L125-L135